components/Typography.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { ReactNode, FC } from 'react';
+
+interface TypographyProps {
+  children: ReactNode;
+  color?: string;
+}
+
+export const TitleXL: FC<TypographyProps> = ({ children }) => (
+  <h1 className="font-poppins text-8xl font-semibold leading-tight capitalize">{children}</h1>
+);
+
+export const TitleL: FC<TypographyProps> = ({ children }) => (
+  <h1 className="font-poppins text-7xl font-semibold leading-tight capitalize">{children}</h1>
+);
+
+export const TitleM: FC<TypographyProps> = ({ children }) => (
+  <h1 className="font-poppins text-6xl font-semibold leading-tight capitalize">{children}</h1>
+);
+
+export const TitleS: FC<TypographyProps> = ({ children }) => (
+  <h1 className="font-poppins text-5xl font-semibold leading-tight capitalize">{children}</h1>
+);
```

Comment:
TitleS to TitleXL should have Eastman font 
You can find the Eastman font in the shared document on Notion